Framework/CurveFitting/inc/MantidCurveFitting/RalNlls/TrustRegion.h

```diff
@@ -19,19 +19,19 @@ void mult_Jt(const DoubleFortranMatrix &J, const DoubleFortranVector &x,
 double evaluate_model(const DoubleFortranVector &f,
                       const DoubleFortranMatrix &J,
                       const DoubleFortranMatrix &hf,
-                      const DoubleFortranVector &d, const nlls_options options,
+                      const DoubleFortranVector &d, const nlls_options &options,
                       evaluate_model_work &w);
 double calculate_rho(double normf, double normfnew, double md,
                      const nlls_options &options);
 void update_trust_region_radius(double &rho, const nlls_options &options,
                                 nlls_inform &inform, NLLS_workspace &w);
-void rank_one_update(DoubleFortranMatrix &hf, NLLS_workspace w);
+void rank_one_update(DoubleFortranMatrix &hf, NLLS_workspace &w);
```

Comment:
The convergence test in `TrustRegionMinimizer::iterate` seems to depend on it. It seems the change here is correct, and it might fix a previously broken minimizer, but I am not really sure.

I think this should be refactored as part of another issue, could you please create one? Also tests should be added, mention in particular to verify what we discussed here.

Apart from that probably :shipit:, but a second look at the minimizers from someone who knows what is going on there would be good.